### ISSUE TITLE
Inline Edits: Hide the  clear button from screen readers

### ISSUE THREAD

**Issue by arodenbeck** [salesforce/design-system-react#1230]
The inline edit component shifts out of edit mode when the input loses focus. Due to this constraint, the clear button cannot take focus. Screen readers are still able to perceive this button in the DOM however. This button should be aria-hidden to prevent screen reader users from perceiving a button they cannot operate.

**Comment by donnieberg** [comment]
I think we're actually going to change the behavior of the Inline Edit so that when you're in edit mode and press Tab, it'll focus the Clear button. And if you press Tab again, then it'll blur the field.

**Comment by interactivellama** [comment]
Should we close this one and open another one with the new spec?

**Comment by arodenbeck** [comment]
Do we have a label we can apply that is something like "waiting on updates from SLDS"? I'm OK with closing this one if that's the case, but depending on how long it is before the SLDS component is updated, I don't want to lose the reason why this component needs updating.

**Comment by interactivellama** [comment]
https://github.com/salesforce/design-system-react/labels/SLDS-action-needed label applied.

**Comment by donnieberg** [comment]
Sorry I should have clarified. The changed focus behavior is something that we can control and it's captured in a different thread. So this issue is a dupe of this one: https://github.com/salesforce/design-system-react/issues/702
